app/views/admins/survey_responses/show.html.erb

```diff
@@ -2,9 +2,15 @@
 <% back_link "Back to Responses", admin_study_survey_survey_responses_path(@survey.study, @survey) %>
 
 <div class="flex gap-y-4">
-  <% @survey_response.responses.each do |r| %>
-    <div class="border bg-slate-50 p-4">
-      <%= r.as_json %>
+  <% if @survey_response.present? && @survey_response.responses.present? %>
+    <% @survey_response.responses.each do |r| %>
+      <div class="border bg-slate-50 p-4">
+        <%= r.as_json %>
```

Comment:
similar to the CSV generation, I'd love to see the columns extracted and shown in the header. We'll probably want to add pagination for this soon, but I'll open a separate ticket for that.